changelog.md

```diff
@@ -0,0 +1,87 @@
+**Changelog**
+====================================
+**v1.1.5 (2021-06-16)**
+------------------------------------
+* Update dependencies
+* Merge branch 'main' of github.com:NUMde/compass-numapp-backend
+* Merge branch 'NUMde:main' into main
+* Merge branch 'main' of https://github.com/mahvaezi/compass-numapp-backend
+* Merge pull request #16 from mahvaezi/main
+(subject_id to subjectId)
+
+**v1.1.4 (2021-06-15)**
+------------------------------------
+* Changing subject_id to subjectId
+* Merge pull request #15 from NUMde/fix_questionnaire_structure
+(Removed unnecessary level for valueCodeableConcept to ensure standard compliance)
+* Debug Configuration for Unit-Tests
+* Merge branch 'NUMde:main' into main
+
+**v1.1.3 (2021-06-14)**
+------------------------------------
+* Removed unnecessary level for valueCodeableConcept to ensure standard compliance
+**v1.1.2 (2021-06-08)**
+------------------------------------
+* Merge pull request #11 from NUMde/generify-tests
+,Generify tests
+
+**v1.1.1 (2021-06-07)**
+------------------------------------
+* Merge pull request #8 from lenzch/add-fhir-config-in-api-file
+,Add fhir config in api file
+* Move calculation of dates in on method
+
+**v1.1.0 (2021-06-03)**
+------------------------------------
+* Made the API doc more specific
```

Comment:
Why does this change trigger a change of the minor version from 0 to 1? i would have expected that this would be v1.0.10. Maybe a short definition on when to increase the major, minor, or patch level would be helpful in the documentation.

Reply:
Maybe [this](https://semver.org/) is something we could refer to

Reply:
I thought we would use the pattern /v([0-9]\.)([0-9]\.)([0-9]). What would you suggest?

Reply:
@mbastian93 the reference is perfect from my point of view. For the regex I would rather use this pattern: `^([0-9]+)\.([0-9]+)\.([0-9]+)`